operator/v1/types_console.go

```diff
@@ -40,9 +44,14 @@ type ConsoleCustomization struct {
 	// Invalid value will prevent a console rollout.
 	DocumentationBaseURL string `json:"documentationBaseURL,omitempty"`
 	//customProductName is the customizable product name that can be set for the web console
+	// +optional
```

Comment:
Should there be symmetry between:

```
Brand
CustomBrand
```
or are we attacked to `CustomProductName`?

Reply:
This will also need better a better comment for docs purposes.

Reply:
I am fine with a name change; I simply used the same name that was used in the console code so we would be consistent